CMakeLists.txt

```diff
@@ -75,11 +75,11 @@ add_definitions(VKFW_NO_SPACESHIP_OPERATOR)
 add_definitions(VKFW_NO_STRING_VIEW)
 add_definitions(VKFW_NO_SPAN)
 
-if (VKFW_INSTALL)
 if(VKFW_BUILD_CXX_MODULE)
     add_subdirectory(CppModule)
 endif()
 
+if(VKFW_INSTALL)
```

Comment:
Squash this commit into the commit that introduces `if(VKFW_BUILD_CXX_MODULE)` please.